### ISSUE TITLE
FileMenu: save project (as) should provide selected project path without ".dir" extension

### ISSUE THREAD

**Issue by Mikrischu** [einDamian/discrete-choice#30]
When you save the project a directory with ".dir" extension is created. When you print the path of the FileDialog it can be shown, that the by view returned path contains already the ".dir" extension.

<img width="912" alt="Bildschirmfoto 2023-07-20 um 13 19 14" src="https://github.com/einDamian/discrete-choice/assets/133114285/2cdda806-ef80-4f0a-a965-53d102152218">
<img width="446" alt="Bildschirmfoto 2023-07-20 um 13 20 02" src="https://github.com/einDamian/discrete-choice/assets/133114285/f1bbec9e-70e9-422c-b9a7-1e270962d13a">

**Comment by alice221927** [comment]
Can you describe the problem more? 
After getting the path and the project's name using file dialog, it returns a path e.g. /home/user/Desktop/Books/name

**Comment by alice221927** [comment]
Ah great. I am still trying to figure out, how the problem is being caused. Sorry if I ask a lot of questions.
If I try to print the path which I get from FileManagementWindow in the View, I receive something like this:
/home/user/Desktop/Books/name 
I don't see the dir-extension here... Could it be due to using different operation system?

**Comment by alice221927** [comment]
As I mentioned above, on linux the path has the right pattern. I'll see what I can change about it. In the end you have to test it on your operating system again.

**Comment by alice221927** [comment]
I used the naive way to solve the problem. before returning the path from FileManagementWindow. save_file(), the string will be checked, if there is a  '.dir' extension at the end of the path and removed it.

I tried different approaches, but they didn't work, as they showed all files for selection (while only directories are expected)
1. Used the options field in getSaveFile() options = QFileMode.ShowDirsOnly
2. Used the option above and set the FileMode to QFileDialog.Directory
3. tried to find different native file dialog with same behavior 
4. Idea: use a custom file dialog, but this is only possible if we use non native file Dialog

According to my researches the native QFileDialog behaviors differently on different operation systems (As native file dialogs do differ on operating systems). 
I suggest we test the program on Windows, once we are done with programming.
